### ISSUE TITLE
Documentation menu on mobile

### ISSUE THREAD

**Issue by ju5t** [blade-ui-kit/blade-ui-kit.com#4]
I was almost hesitant to raise a bug because you've done such an amazing job. But the documentation menu is taking up a lot of space on mobile. I've made a screenshot on my desktop but you get the gist of it.

Either way, thanks a lot for all the work you put into this!

![image](https://user-images.githubusercontent.com/3635751/92020761-f74ac980-ed58-11ea-8666-f0cca1bbd984.png)

**Comment by driesvints** [comment]
Yeah definitely! I didn't get to that before the launch but it's on my list. Thanks for raising the issue.
